www/components/Input.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import styles from "../styles/LogReg.module.css";
+import PropTypes from "prop-types";
+
+const Input = ({
+  label,
+  placeholder,
+  onChange,
+  value,
+  size,
+  required,
+  varient,
+}) => {
+  Input.propTypes = {
```

Comment:
Do `propType` specs work inside of the component? The docs show something like this:
```js
import PropTypes from 'prop-types';

class Greeting extends React.Component {
  render() {
    return (
      <h1>Hello, {this.props.name}</h1>
    );
  }
}

Greeting.propTypes = {
  name: PropTypes.string
};
```

Reply:
They do, I think it probably makes more sense to have them outside though if that's what the docs say. Maybe some of the errors I was having were due to that